src/applications/ivc-champva/shared/components/fileUploads/MissingFileList.jsx

```diff
@@ -75,46 +141,38 @@ export default function MissingFileList({
   return (
     <div>
       <h3 className="vads-u-font-size--h4">{title || ''}</h3>
-      <p>
-        <strong>{description || ''}</strong>
-      </p>
-      {wrapped.map((entry, idx) => {
+      <p>{description || ''}</p>
+      {wrapped.map((entry, outerIndex) => {
         if (
           entry?.missingUploads.filter(f => f.required === subset).length === 0
         )
           return <></>;
-        const entryName = `${entry[nameKey].first || ''}`;
+        const entryName = `${entry[nameKey]?.first ?? ''}`;
         return (
           <div key={`${entryName}-${subset}`}>
-            {showNameHeader ? <strong>{entryName}</strong> : null}
+            {showNameHeader ? <h3>{entryName}</h3> : null}
             <ul
               style={
                 !disableLinks && !showFileBullets
                   ? { listStyleType: 'none' }
                   : { listStylePosition: 'inside' }
               }
             >
-              {entry.missingUploads?.map((file, index) => {
-                const fn =
-                  fileNameMap?.[file.name] ?? makeHumanReadable(file.name);
-                return file.required === subset &&
-                  (disableLinks ? file.uploaded === false : true) ? (
-                  <li
-                    key={file.name + file.uploaded + index}
-                    className="vads-u-margin-y--1"
-                  >
-                    {!disableLinks ? (
-                      <>
-                        {showFileBullets ? fn : null}
-                        <br />
-                        {alertOrLink(file, entryName, idx, fileNameMap)}
-                      </>
-                    ) : (
-                      `${entryName}’s ${fn}`
-                    )}
-                  </li>
-                ) : null;
-              })}
+              {entry.missingUploads?.map((file, innerIndex) =>
+                // some prop-drilling happening here, but that's to
+                // keep this function slightly higher-level.
+                buildListItems({
+                  fileNameMap,
+                  file,
+                  subset,
+                  disableLinks,
+                  showFileBullets,
+                  entryName,
+                  innerIndex,
+                  outerIndex,
+                  listItemShowNamePrefix,
+                }),
+              )}
```

Comment:
Pulled this markup out into the `buildListItems` function so that things could be a tad more modular.